packages/core/android/src/main/java/com/datadog/reactnative/DdRum.kt

```diff
@@ -33,7 +35,14 @@ class DdRum(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(
      */
     @ReactMethod
     fun startView(key: String, name: String, context: ReadableMap, timestampMs: Double, promise: Promise) {
-        nativeInstance.startView(key, name, context.toHashMap(), timestampMs.toLong())
+        val attributes = context.toHashMap().toMutableMap().apply {
```

Comment:
I'm doing a lot of `.toHashMap().toMutableMap()`, is there a simpler way to achieve this?

Reply:
I don't think you will find a better way, if you want to modify that `Map` you need to convert it to a mutable one.

Reply:
we can write an extension method `ReactApplicationContext#toMutableMap`